src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -0,0 +1,96 @@
+
+The following widgets can be found in the library:
+
+- **Distribution Duration**: A multi-y axis chart visualizing the distribution of transaction duration across various percentiles (p50, p75 and p95)
+- **High Throughput Transactions**: The top 5 transactions with the largest number of events
+- **LCP by Country**: World map showing the p75 of page load times for each country
+- **Miserable Users**: The total number of unique users who have experienced slow transactions (transaction duration greater than 1200ms)
+- **Slow vs Fast Transactions**: Bar chart comparing the percentage of transactions that are over 300ms (slow) and under 300ms (fast)
+- **Latest Unresolved Issues**: A table of unresolved issues ordered by the most recently seen issues
+- **Top Unhandled Error Types**: The top 5 most frequently encountered unhandled errors
+- **Users Affected by Errors**: A comparison of the total number of errors and the number of unique users affected by the errors
+
+You can change the title, queries, fields, visualization types and sort order of these prebuilt widgets to suit your use case by clicking on the context menu on the widget and selecting "Edit Widget".
+
+## Example Use Cases
+
+### Performance across release versions
+
+To monitor how your app is performing on a certain release, you can modify the **Distribution Duration** widget by adding the release version to the search condition.
+
+- Search condition 1: `!event.type:error release.version:{version}`
+
+To compare performance with another release version, you can add another query by clicking the "Add Query" button and querying for a different release version.
+
+- Search condition 2: `!event.type:error release.version:{another_version}`
+
+You can also compare performance before/and after a certain version with the following conditions:
+
+- Search condition 1: `!event.type:error release.version:<{version}`
+- Search condition 2: `!event.type:error release.version:>={version}`
+
+
+### Response thresholds
+
+The **Distribution Duration** widget shows how the spread of transaction duration times. This visualization can be modified to show if transaction durations are within certain target thresholds and present them as "satisfactory", "tolerable" or "frustrating" transactions. Change the "Visualization Display" to "Area Chart". Area charts stack results and are more appropriate for results that are cummulative.
+
+- Visualization Display: `Area Chart`
+
+Set one of the y-axis values to `count()` and remove the other two axes.
+
+- Y-Axis: `count()`
+
+To the first query, add the search condition for satisfactory transaction duration (this example uses 300ms as the satisfactory response threshold).
+
+- Search Condition 1: `!event.type:error transaction.duration:<300`
```

Comment:
Just keeping this consistent with the first example since this doesn't represent any UI text. 
```suggestion
- Search condition 1: `!event.type:error transaction.duration:<300`
```